com.github.lxgr_linux.pokete.yml

```diff
@@ -0,0 +1,69 @@
+app-id: com.github.lxgr_linux.pokete
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pokete
+
+finish-args:
+  # Store save files and logs
+  - "--filesystem=xdg-cache/pokete:create"
+  # Trading and update checking
+  - "--share=network"
+
+cleanup:
+  - "/assets"
+  - "/.github"
+  - "/*.md"
+  - "/.*"
+  - "/Pipfile*"
+
+modules:
+  # Dependency of scrap engine; generated with flatpak-pip-generator
```

Comment:
just a trick: we usually *include* the json generated by flatpak-pip-generator for ease of maintainance: you can just regenerate the file

(It's not blocking, but this is a common practice)